media/js/newsletter/newsletter.es6.js

```diff
@@ -68,6 +68,14 @@ const NewsletterForm = {
                 });
             }
         }
+
+        // Glean
+        if (typeof window.Mozilla.Glean !== 'undefined') {
+            window.Mozilla.Glean.pagePing({
+                label: 'newsletter-sign-up-success',
+                type: newsletters.join(', ')
```

Comment:
For GA4 I am logging each newsletter sign up as a separate event (so a form that subscribes to 2 newsletters triggers 2 events). My thinking was that if we are interested in looking at sign-ups to the X newsletter it might be easier to filter for them if we're not also trying to include sign-ups to the X,Y newsletter. OTOH I have no idea what I'm doing 🎢  Maybe you want to copy that here or maybe you want to keep it this way and see how my implementation works.

Reply:
Hmm, good question! I've tried to think through the pros / cons of this, and I'm inclined to leave it as a single event because:

- It makes it slightly easier to query / report on who subscribes to both newsletters, as opposed to only one.
- It ties one user action to one event.
- It's slightly less data over the wire.

I think filtering for only one newsletter should be easy enough still, as you just need to make sure the field contains the slug, as opposed to do a strict comparison.